command/alloc_status.go

```diff
@@ -374,14 +374,14 @@ func formatAllocNomadServiceChecks(allocID string, client *api.Client) string {
 	} else if len(statuses) == 0 {
 		return ""
 	}
-	results := []string{"Service|Task|Name|Mode|Status"}
+	results := []string{"Service|Task|Name|Mode|Status|Notes"}
 	for _, status := range statuses {
 		task := "(group)"
 		if status.Task != "" {
 			task = status.Task
 		}
-		// check | group | mode | status
-		s := fmt.Sprintf("%s|%s|%s|%s|%s", status.Service, task, status.Check, status.Mode, status.Status)
+		// check | group | mode | status | notes
+		s := fmt.Sprintf("%s|%s|%s|%s|%s|%s", status.Service, task, status.Check, status.Mode, status.Status, status.Notes)
```

Comment:
this unit test fails, the `status.Notes` field is empty

Reply:
Nice, thanks for checking. I just pushed https://github.com/hashicorp/nomad/pull/22397/commits/bf4324cd6f23047c0fbe2fd799235261fa2068bb to try to address this, I hope it helps. Running `go test` in this package doesn't work on my laptop, even on the `main` branch.

Did `NSD` in `TestAllocStatusCommand_NSD_Checks` stand for `Nomad Service D...iscovery?`

Reply:
Should we really report notes in a table like this at all? Won't the note push off the edge of the terminal most of the time?

Reply:
We intend to "abuse" this field to store structured information, so having it in the CLI table could definitely come in handy at some point for us. However, that's just our use case and we don't strictly need it, so I can remove that part from the PR if you want

Reply:
Also the test I wrote is not working, so let me know what you guys decide so I can fix it or delete it

Reply:
Hey @nicoche, I took another look at this and here's the bit I don't understand. You're implementing `notes` field support for Consul service health checks. But this command, `nomad alloc status`, will not display details about services if they aren't managed by Nomad (service block `provider` field must be set to `nomad`). This is also what causes unit test issues: mock Consul service jobs with notes won't change the output of `nomad alloc status`, and for mock Nomad service jobs, the `notes` field will always be empty.

Reply:
if you feel like it, you may [add](https://github.com/hashicorp/nomad/blob/953bfcc31e7e283acb44ea04c3b288bd5d65dd65/client/serviceregistration/nsd/nsd.go#L351) the `notes` field to nomad managed services, but I think this is maybe out of scope and, more importantly, not what you want?

Reply:
Hey guys! I simply removed that chunk 🙂